docs/HowTo/clusterconnectivity/aiven-kafka-connect-cluster-ssl-protocol.md

```diff
@@ -52,7 +52,7 @@ Kafka Connect service with Klaw using SSL protocol:
    the copy icon that is available on the right-hand side of each
    cluster row.
 
-8. In the `application.properties` file for [cluster-api] (klaw/cluster-api/src/main/resources) module, configure Aiven for Apache Kafka Connect credentials copied from Aiven console:
+8. In the `application.properties` file for [cluster-api] (Klaw/cluster-api/src/main/resources) module, configure Aiven for Apache Kafka Connect credentials copied from Aiven console:
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?